apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRoutes.kt

```diff
@@ -60,6 +61,20 @@ fun Route.trygdetid(
             }
         }
 
+        post("overstyr") {
+            withBehandlingId(behandlingKlient) {
```

Comment:
Kunne denne tatt inn trygdetidid og behandlingid? Da blir det lettere for meg senere, når trygdetiden er koblet på person og behandling :D

Reply:
Jepp, perfekt!